archutils/epoll.go

```diff
@@ -1,4 +1,4 @@
-//+build !arm64,linux
+// +build linux,!aarch64
```

Comment:
@cyphar while you're at it, this should have an extra line or else it seems like it's a package comment? I even wonder how it could have been working so far...